### PR TITLE
UI refresh

### PR DESCRIPTION
**Pull request by zarino**
This brings the generic UI improvements from https://github.com/mysociety/mapit.mysociety.org/issues/29 into mapit core.

Works really well with the latest version of https://github.com/mysociety/mapit.mysociety.org/pull/32, but needs testing in the other mapit cobrands (like global, and italy).

### CODE REVIEW

File: mapit_it/templates/mapit/base.html [hunk 1]
```diff
@@ -1,88 +1,4 @@
-{% load static from staticfiles %}
+{% extends "mapit/base.html" %}
```
Comment (dracos) [inline mapit_it/templates/mapit/base.html]: Drive-by: This won't work I'm afraid because it's the same template being overridden.
Put the description in a separate template and then override that?

Comment (zarino) [inline mapit_it/templates/mapit/base.html]: The only reason I'm overriding `base.html` here is to insert the translated page description. Could we just upload the translation via Transifex instead, and then the override wouldn't even be required?

Comment (dracos) [inline mapit_it/templates/mapit/base.html]: It already is `trans`, so yes,i don't think this is necessary at all.

File: mapit/static/mapit/sass/modules/_footer.scss [hunk 2]
```diff
@@ -1,74 +0,0 @@
...
-
-  hr {
-    border-width: 1px 0 0 0;
-  }
-
-  p {
-    margin-top: 0;
-  }
-
-  .mapit-copyright {
```
Comment (dracos) [inline mapit/static/mapit/sass/modules/_footer.scss]: Could we keep the copyright text being a bit smaller? :)

File: mapit/static/mapit/sass/modules/area_view.scss [hunk 3]
```diff
@@ -1,94 +0,0 @@
...
-        clear: none;
-        float: right;
-        padding: 1em 0.625em 2.5em 0.625em;
-    }
-*/
-
-    @media (min-width: 60em) {
-        float: right;
-        clear: none;
-        width: 28.75em;
```
Comment (dracos) [inline mapit/static/mapit/sass/modules/area_view.scss]: I quite liked the two columns of text output on an area page.

File: mapit/static/mapit/sass/_forms.scss [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    width: auto;
+    margin: 0 0.5em;
+    vertical-align: middle;
+}
+
+.homepage-try {
+    input[type="text"] {
+        @extend .form-control;
+        @extend .form-control--inline;
+        text-transform: uppercase;
```
Comment (dracos) [inline mapit/static/mapit/sass/_forms.scss]: On Global, "LATITUDE,LONGITUDE" does not fit.

File: mapit/static/mapit/sass/modules/area_list.scss [hunk 5]
```diff
@@ -1,46 +0,0 @@
...
-            margin: 0;
-            font-size: 0.75em;
-        }
-    }
-}
-
-// Creates area_level_1 to area_level_10
-// optionally add to list items to add indentation suggesting hierarchy
-
-@while $area_level > 0 {
```
Comment (dracos) [inline mapit/static/mapit/sass/modules/area_list.scss]: It would be a shame to lose this (which indents the area lists on e.g. http://global.mapit.mysociety.org/point/4326/8.55,47.366667.html).

**Comment by dracos** [comment]
One other thing, do you think the line between sections of the docs might be a bit faint? I can only just see it on my monitor :)

File: mapit/static/mapit/sass/modules/_footer.scss (hunk 2 above)
Comment (zarino) [inline mapit/static/mapit/sass/modules/_footer.scss]: Done!

File: mapit/static/mapit/sass/modules/area_view.scss (hunk 3 above)
Comment (zarino) [inline mapit/static/mapit/sass/modules/area_view.scss]: Done!

File: mapit/static/mapit/sass/_forms.scss (hunk 4 above)
Comment (zarino) [inline mapit/static/mapit/sass/_forms.scss]: Done!

File: mapit/static/mapit/sass/modules/area_list.scss (hunk 5 above)
Comment (zarino) [inline mapit/static/mapit/sass/modules/area_list.scss]: Done!
